### DISCUSSION TITLE
Different actions when using modifier keys

### DISCUSSION THREAD

**Discussion by jh12z** [Ulauncher/Ulauncher#1161] [Ideas]
I'm trying to find if it's possible to do different actions when selecting an item from the list based on any modifier keys pressed.
I come from using Spotlight in MacOS and there you can:
- Press <kbd>Return</kbd> which is the default action for the item (open the file, application, ...)
- Press <kbd>Command</kbd> + <kbd>Return</kbd> will open the parent folder of the selected item
- Press <kbd>Option</kbd> + <kbd>Return</kbd> will take the current search criteria and create a Finder search window out of it

Is it possible to create an extension that can handle those scenarios. I think it'd be great to provide that functionality.

I'm using https://github.com/friday/ulauncher-locate extension which is fantastic but would like to add that additional functionality to it.

**Comment by friday** [comment]
It's possible with Alt + Enter only, if the extension enables it. But this is not very "discoverable" and feels unsafe imo. See #326 for one proposal similar to how I remember it from Alfred (I haven't used Mac in a really long time now though).
